src/gdb.ts

```diff
@@ -42,9 +36,8 @@ class GDBDebugSession extends MI2DebugSession {
 		this.sendResponse(response);
 	}
 
-	protected launchRequest(response: DebugProtocol.LaunchResponse, args: LaunchRequestArguments): void {
-		this.miDebugger = new MI2(args.gdbpath || "gdb", ["-q", "--interpreter=mi2"], args.debugger_args);
-		this.initDebugger();
+
+	private initiliaseDefaultValueArgs(args: CommonRequestArguments, attach: boolean) {
```

Comment:
The other parts of the program spell it initialize instead of initialise, can you keep it consistent and change it to the american spelling?